src/tbp/monty/frameworks/utils/spatial_arithmetics.py

```diff
@@ -40,11 +40,11 @@ def rot_mats_to_quats(rot_mats, invert=False):
         Quaternions
     """
     quats = []
-    for rot_mat in rot_mats:
-        rot_mat = Rotation.from_matrix(rot_mat)
+    for mat in rot_mats:
```

Comment:
Can we give this a more expressive name instead of a less expressive name? Something like `rotation_matrix`?